### PR TITLE
Feature/course

### PR DESCRIPTION
**Pull request by MekalaArunKumar16**

### CODE REVIEW

File: src/assets/images/video.svg
Comment (jrTilak) [inline src/assets/images/video.svg:1]: The image and the play button are different. Have you also added the play button on top of this video?

File: src/pages/CoursePage/Ui/Curriculum.tsx [hunk 1]
```diff
@@ -0,0 +1,247 @@
+import React, { useState, useRef } from 'react'
+import play from '../../../assets/images/arrow-right.svg'
+
+const Curriculum: React.FC = (): JSX.Element => {
+  const Playlist: React.FC = () => {
```
Comment (jrTilak) [inline src/pages/CoursePage/Ui/Curriculum.tsx]: Why there is component inside a component, Try to make another component in separate file.

**Comment by jrTilak** [comment]
### I guess you have used tick image instead of thumbnail image.

image

**Comment by jrTilak** [comment]
image

**Comment by jrTilak** [comment]
Not proper responsive in 1000-1400px

**Review by aakashk1905 — APPROVED** [review]
Add Route for dashboard again

**Review by jrTilak — APPROVED** [review]

**Review by neon-lab-dev — APPROVED** [review]

**Pull request merged**